src/dl_node.hpp

```diff
@@ -41,20 +41,23 @@ class DLNode : public Node {
     std::shared_ptr<ModelInstance> model;
     std::unique_ptr<NodeStreamIdGuard> nodeStreamIdGuard;
     std::unique_ptr<ModelInstanceUnloadGuard> modelUnloadGuard;
+    std::optional<uint32_t> demultiplexCount;
 
 public:
     DLNode(const std::string& nodeName, const std::string& modelName, std::optional<model_version_t> modelVersion,
         ModelManager& modelManager,
-        std::unordered_map<std::string, std::string> nodeOutputNameAlias = {}) :
+        std::unordered_map<std::string, std::string> nodeOutputNameAlias = {}, uint32_t demultiplyCount = 0) :
         Node(nodeName),
         modelName(modelName),
         modelVersion(modelVersion),
         modelManager(modelManager),
-        nodeOutputNameAlias(nodeOutputNameAlias) {
+        nodeOutputNameAlias(nodeOutputNameAlias),
+        demultiplexCount(demultiplyCount ? std::optional<uint32_t>(demultiplyCount) : std::nullopt) {
     }
 
     Status execute(session_key_t sessionKey, PipelineEventQueue& notifyEndQueue) override;
 
+    using Node::fetchResults;
```

Comment:
Check but I think in the end this using is not needed - just the one in mocked DLNode.